lib/trento/activity_logging/policy.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Trento.ActivityLog.Policy do
+  @moduledoc """
+  Policy for the activity log.
+
+  User with the ability all:all/all:user can see user management activity log entries.
+  User with all other abilities can not see any user management activity log entries.
+  """
+
+  @behaviour Bodyguard.Policy
+  alias Trento.Support.PolicyHelper, as: Helper
```

Comment:
Please don't alias as Helper, PolicyHelper is more readable and in context with the file